### ISSUE TITLE
Feature request: The app should be able to remember the last playback position

### ISSUE THREAD

**Issue by stephankellermayr** [rasmuslos/AmpFin#53]
I have just bought the app in the App Store and unfortunately I have noticed that an important feature (for me) is missing:  
It would be fantastic if the app could remember the last playback position in case it closes or is interrupted in some other way.

This feature is extremely important, especially for audiobooks.
They usually come in rather unwieldy files, and finding the last position every time is extremely tedious.

Thanks anyway for this great app! A feature that I sorely missed in Jellyfin.

---

**Edit:** To put it more precisely:

The requested feature would be that the app can remember the playback position within a playlist.

For example:
1. you create one playlist per audiobook
2. you start playing the playlist
3. when closing the app or switching to another playlist, the last position of the playlist is saved
4. as soon as the playlist is opened again, the app starts playing again at the exact position previously saved.

(Summarized in layman's terms)

**Comment by rasmuslos** [comment]
Hey,

audiobooks are out of AmpFin's scope. The app is purely intended for music playback. I found Jellyfin's support for audiobooks lacking, so I use https://github.com/advplyr/audiobookshelf a media server that is specifically made for audiobooks and podcasts. https://github.com/rasmuslos/ShelfPlayer is AmpFin's sibling for audiobooks and podcasts, but it does not support Jellyfin as a backend, only Audiobookshelf.

**Comment by stephankellermayr** [comment]
@rasmuslos OK, I see.

And apart from Audiobooks: Don't you think it would still be a helpful feature if the app could remember the playback position in a music playlist?

**Comment by rasmuslos** [comment]
I don't know any other music app that remembers your position inside a playlist and I don't think it would be particularly useful because I imagine most people shuffle their playlists.

Audiobookshelf is basically the same as Jellyfin but for audiobooks, so it does not work with Jellyfin but does a way better job than Jellyfin for audiobooks.

**Comment by plexyxd** [comment]
+1 for this feature. I know multiple other apps support such a feature (Apple Music, YouTube Music, Nintendo Music, etc.), and I would certainly appreciate this one.

**Comment by stephankellermayr** [comment]
btw: Since I still miss that fetaure (and I don't seem to be the only one), I switched from AmpFin to [Flacbox](https://www.everappz.com/flacbox). It doesn't work as a front end for Jellyfin, but it reads the data directly from a cloud and even remembers the last playback position within an album.
